cmd/internal/shared/applicationserver/config.go

```diff
@@ -76,4 +76,7 @@ var DefaultApplicationServerConfig = applicationserver.Config{
 			Workers: 16,
 		},
 	},
+	Formatters: applicationserver.FormattersConfig{
+		MaxScriptSize: 4096,
```

Comment:
I think this should be `MaxParameterLength` to be consistent with the naming in AS